wordpress-mobile/WordPress-Android
# Update media picker selector UI

Fixes #12654 

This PR updates part of the new media picker UI - the selectors - based on the designs.
These are the new states: 
Single selection:
<img width="471" alt="Screenshot 2020-09-03 at 15 23 42" src="https://user-images.githubusercontent.com/1079756/92120637-9aebb680-edf9-11ea-97a0-69269518488e.png">
Multi selection:
<img width="465" alt="Screenshot 2020-09-03 at 15 23 16" src="https://user-images.githubusercontent.com/1079756/92120649-9de6a700-edf9-11ea-81a4-e445944bbbb3.png">
 

To test:
- Go to Media library/+/Choose from device
- Try selecting a few things 
- Confirm the UI looks OK

To test 2:
- Go to GB editor/Image block/Add media/Choose from device
- Try selecting an item
- Confirm the UI looks OK 

PR submission checklist:
